app/models/account.rb

```diff
@@ -63,24 +63,24 @@ class Account < ApplicationRecord
   has_many :reports
   has_many :targeted_reports, class_name: 'Report', foreign_key: :target_account_id
 
-  scope :remote, -> { where.not(domain: nil) }
-  scope :local, -> { where(domain: nil) }
-  scope :without_followers, -> { where('(select count(f.id) from follows as f where f.target_account_id = accounts.id) = 0') }
-  scope :with_followers, -> { where('(select count(f.id) from follows as f where f.target_account_id = accounts.id) > 0') }
-  scope :expiring, ->(time) { where(subscription_expires_at: nil).or(where('subscription_expires_at < ?', time)).remote.with_followers }
-  scope :silenced, -> { where(silenced: true) }
-  scope :suspended, -> { where(suspended: true) }
-  scope :recent, -> { reorder(id: :desc) }
-  scope :alphabetic, -> { order(domain: :asc, username: :asc) }
-  scope :by_domain_accounts, -> { group(:domain).select(:domain, 'COUNT(*) AS accounts_count').order('accounts_count desc') }
+  scope :remote, (-> { where.not(domain: nil) })
+  scope :local, (-> { where(domain: nil) })
+  scope :without_followers, (-> { where('(select count(f.id) from follows as f where f.target_account_id = accounts.id) = 0') })
+  scope :with_followers, (-> { where('(select count(f.id) from follows as f where f.target_account_id = accounts.id) > 0') })
+  scope :expiring, (->(time) { where(subscription_expires_at: nil).or(where('subscription_expires_at < ?', time)).remote.with_followers })
+  scope :silenced, (-> { where(silenced: true) })
+  scope :suspended, (-> { where(suspended: true) })
+  scope :recent, (-> { reorder(id: :desc) })
+  scope :alphabetic, (-> { order(domain: :asc, username: :asc) })
+  scope :by_domain_accounts, (-> { group(:domain).select(:domain, 'COUNT(*) AS accounts_count').order('accounts_count desc') })
 
   delegate :email,
-    :current_sign_in_ip,
-    :current_sign_in_at,
-    :confirmed?,
-    to: :user,
-    prefix: true,
-    allow_nil: true
+           :current_sign_in_ip,
```

Comment:
I'm generally skeptical of commits/changes that JUST fix rubocop warnings w/out also improving readability ... I'd put these indentation ones in that category.